src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/RejectApplications.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@model SFA.DAS.LevyTransferMatching.Web.Models.Pledges.RejectApplicationsViewModel
+
+@{
+    ViewData["Title"] = "Reject Applications";
+}
+
+<a href="@Url.Action("Applications","Pledges", new { EncodedAccountId = Model.EncodedAccountId, EncodedPledgeId = Model.EncodedPledgeId })" class="govuk-back-link">Back</a>
+
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content">
+
+    <div class="govuk-width-container">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <h1 class="govuk-heading-xl">Rejecting the apprenticeship application(s)</h1>
+             
+                <form method="post">
+                    <section class="das-highlight">
+                        <p class="govuk-heading-s">Applications for rejection</p>
+                        <ul class="govuk-list govuk-list--bullet">
+                            @foreach (var applicantionName in Model.DasAccountNames)
+                            {
+                                <li class="application_rejection" style="display: list-item;">@applicantionName</li>
+                            }
+                        </ul>
+                    </section>
+
+                    <div class="govuk-form-group" >
+                        <fieldset class="govuk-fieldset" aria-describedby="contact-hint">
+                            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                                <h2 class="govuk-heading-m no-margin-bottom">What would you like to do?</h2>
+                            </legend>
+                            <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
+                                <div class="govuk-radios__item">
+                                    <input class="govuk-radios__input" id="reject_application_reject" name="RejectConfirm" type="radio" value="true">
+                                    <label class="govuk-label govuk-radios__label" for="reject_application_reject">Reject the application(s)</label>
+                                </div>
+                                <div class="govuk-radios__item">
+                                    <input class="govuk-radios__input" id="reject_application_cancel" name="RejectConfirm" type="radio" value="false">
+                                    <label class="govuk-label govuk-radios__label" for="reject_application_cancel">Cancel, return to applications</label>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </div>
+
+                    <button class="govuk-button" data-module="govuk-button" id="reject_application" type="submit">Continue</button>
```

Comment:
If I fail to give an answer to the confirmation question (either Yes or No), I don't get a validation error but am returned to the previous page. The user ought to see an error message.

Reply:
I couldn't able to find a way to trigger those Fluent Validator messages. It only shows a message "There is a problem" but we want to show the actual message which is linked with the validator rule. And I cannot able to figure out why it's not displaying the message which is defined in the validator "Rule". Will need some help with this.

Reply:
The error messages are now working for these validators.